manifests/server/config_entry.pp

```diff
@@ -3,21 +3,16 @@
 # @param ensure Removes an entry if set to 'absent'.
 # @param key Defines the key/name for the setting. Defaults to $name
 # @param value Defines the value for the setting.
+# @param comment Defines the comment for the setting, ommit the #, it's added by default.
 # @param path Path for postgresql.conf
 #
 define postgresql::server::config_entry (
-  Enum['present', 'absent']                               $ensure = 'present',
-  String[1]                                               $key    = $name,
-  Optional[Variant[String[1], Numeric, Array[String[1]]]] $value  = undef,
-  Variant[Boolean, String[1]]                             $path   = false
+  Enum['present', 'absent']                               $ensure  = 'present',
+  String[1]                                               $key     = $name,
+  Optional[Variant[String[1], Numeric, Array[String[1]]]] $value   = undef,
+  Optional[String[1]]                                     $comment = undef,
+  Variant[Boolean, String[1]]                             $path    = $postgresql::server::postgresql_conf_path
```

Comment:
Should this be a Boolean or String or a Stdlib::AbsoluteUnixPath?

Reply:
I think this should be string now as earlier its was expecting either bool or string but now we have remove below code 
```
$target = $path ? {
    false   => $postgresql_conf_path,
    default => $path,
  }
```
so should be string?